.github/ISSUE_TEMPLATE/joint-evaluation.md

```diff
@@ -1,7 +1,7 @@
 ---
-name: Security Assessment
-about: To request a security assessment or track progress on active assessment
-title: "[Assessment] Project Name"
+name: Joint evaluation
+about: To request a joint evaluation or track progress on active evaluation
+title: "[Joint Evalution] Project Name"
 labels: assessment
```

Comment:
Do we still want to label them as assessment?

Reply:
@pragashj until #479 and #488 are merged in, it should still be called 'assessment'